Formula/zookeeper.rb

```diff
@@ -38,7 +38,9 @@ def shim_script(target)
 
   def default_zk_env
     <<-EOS.undent
-      export ZOOCFGDIR="#{etc}/zookeeper"
+      if [ -z ${ZOOCFGDIR+x} ]; then
```

Comment:
@ddillinger Looking at the information in that linked StackOverflow, it seems that neither really apply strongly here. I'd also argue that trying to discern between unset and `""` would be the opposite behavior of what we want. `""` is not a valid UNIX path as far as I'm aware.